hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -125,6 +125,15 @@ object DataSourceReadOptions {
       + HollowCommitHandling.USE_TRANSITION_TIME + "`, will use instant's "
       + "`stateTransitionTime` to perform comparison.")
 
+  /* TODO: this is a replacement of HoodieIncrSourceConfig.NUM_INSTANTS_PER_FETCH
+      to make incremental instant limit available in spark incremental relations
+      Need to link HoodieIncrSourceConfig.NUM_INSTANTS_PER_FETCH to this correctly and add doc*/
+  val INCREMENTAL_LIMIT: ConfigProperty[String] = ConfigProperty
+    .key("hoodie.streamer.source.hoodieincr.num_instants")
```

Comment:
Spark should not be aware of this config.  We can pass down the correct instant range without using the `num_instants` config.

Reply:
I think this makes sense, will remove this config and its usages